src/compiler/config/test/validate-dev-server.spec.ts

```diff
@@ -276,4 +276,23 @@ describe('validateDevServer', () => {
     const { config } = validateConfig(inputConfig, mockLoadConfigInit());
     expect(config.devServer.prerenderConfig).toBe(wwwOutputTarget.prerenderConfig);
   });
+
+  describe('pingRoute', () => {
+    it('should default to /ping', () => {
+      const { config } = validateConfig(inputConfig, mockLoadConfigInit());
+      expect(config.devServer.pingRoute).toBe('/ping');
+    });
+
+    it('should set user defined pingRoute', () => {
+      inputConfig.devServer = { ...inputDevServerConfig, pingRoute: '/my-ping' };
+      const { config } = validateConfig(inputConfig, mockLoadConfigInit());
+      expect(config.devServer.pingRoute).toBe('/my-ping');
+    });
+
```

Comment:
Can we add a test for `pingRoute`s without a leading '/'?
```suggestion

    it('should prefix pingRoute with a "/"', () => {
      inputConfig.devServer = { ...inputDevServerConfig, pingRoute: 'my-ping' };
      const { config } = validateConfig(inputConfig, mockLoadConfigInit());
      expect(config.devServer.pingRoute).toBe('/my-ping');
    });
```